Serpent/Serpent/Classes/Extensions/AlamofireExtension.swift

```diff
@@ -34,16 +34,13 @@ public extension Parser {
                     return .failure(NSError(domain: "Serializable.Parser", code: 2048, userInfo: [ NSLocalizedDescriptionKey : "Parsing block failed!", "JSONResponse" : value]))
                 }
                 
-            case let .failure(error):
-                if let data = data , data.count != 0 {
-                    //TODO; Find out what to do with error
-//                    var userInfo = error.userInfo
-//                    userInfo["ResponseString"] = String(data: data, encoding: String.Encoding.utf8)
-//                    let newError = NSError(domain: error.domain, code: error.code, userInfo: userInfo)
-                    return .failure(error)
-                }
+            case .failure(_):
```

Comment:
I think it would be nice to use the error provided here and also put it into the NSError that we return - more information is always better.